python/tank/templatekey.py

```diff
@@ -66,6 +76,27 @@ def __init__(self,
         if not all(self.validate(choice) for choice in self.choices):
             raise TankError(self._last_error)
     
+    @property
+    def choices(self):
+        """
+        :returns:    List of choices available for this key
+        """
+        return self._choices.keys()
+    
+    @property
+    def choice_labels(self):
```

Comment:
I see now that it is more complicated than I thought - choices can now be both annotated and not.
What I meant is that we could potentially do something like this:

```
# old way
> my_template_key.choices
['ma', 'mb']

# new dict style lookup
> my_template_key.labelled_choices
{'ma': 'Maya Ascii', 'mb': 'Maya Binary'}

# or when you don't have any values
> my_template_key.labelled_choices
{'ma': None, 'mb': None}
```

This feels perhaps a little simpler than `get_choice_label` and `choice_labels` - but not sure. Also, probably need to come up with a better name. Hope that makes more sense. Up to you what you feel is better/nicer!